desc/vmec.py

```diff
@@ -242,8 +248,9 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         NFP = eq.NFP
         M = eq.M
         N = eq.N
-        M_nyq = M + 4
-        N_nyq = N + 2 if N > 0 else 0
+        M_nyq = M + 4 if M_nyq is None else M_nyq
+        N_nyq = N + 2 if N_nyq is None else N_nyq
+        N_nyq = 0 if int(N) == 0 else N_nyq
```

Comment:
Do we want to force this even if the user purposely inputs a finite value for `N_nyq`? If that shouldn't be allowed, it would be better to throw an error or warning rather than secretly overwrite the user input.

Reply:
hm, I can throw a warning, if N is zero it makes no sense to have N_nyq, so I think a warning is ok. I will make that change